continuum-auth/src/use-cases/login.test.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response } from 'express';
+import { Login } from './login';
+import config from '../config';
+
+describe('login', () => {
+    it.only('redirects', () => {
+        const request = ({
+            params: { token: 'token' },
+        } as unknown) as Request;
+        const response = ({
+            redirect: jest.fn(),
+        } as unknown) as Response;
+
+        config.auth.login_redirect_url = 'http://login_redirect_url';
```

Comment:
This doesn't seem to update the value. Getting the following error when running the test:

```
➜  continuum-auth git:(309-continuum-auth-tests) ✗ yarn run test ./src/use-cases/login.test.ts
yarn run v1.19.1
$ jest ./src/use-cases/login.test.ts
 FAIL  src/use-cases/login.test.ts
  login
    ✕ redirects (6ms)

  ● login › redirects

    expect(jest.fn()).toHaveBeenCalledWith(...expected)

    Expected: "http://login_redirect_url"
    Received: "undefined:undefined/submit"

    Number of calls: 1

      16 |         Login(request, response);
      17 |         expect(response.redirect).toHaveBeenCalledTimes(1);
    > 18 |         expect(response.redirect).toHaveBeenCalledWith('http://login_redirect_url');
         |                                   ^
      19 |     });
      20 | });
      21 | 

      at Object.<anonymous> (src/use-cases/login.test.ts:18:35)

{"level":30,"time":1572425268783,"pid":53884,"hostname":"Nathans-MBP-2","service":"continuum-auth","realm":"domai
n","msg":"loginRedirect {\"login_redirect_url\":\"undefined:undefined/submit\"}","v":1}
Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        2.313s
Ran all test suites matching /.\/src\/use-cases\/login.test.ts/i.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```